app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -405,6 +444,20 @@ class SamplesHeatmapVis extends React.Component {
             }}
           />
         )}
+        <div
+          className={cx(
+            cs.bannerContainer,
+            this.state.displayControlsBanner ? cs.show : cs.hide
+          )}
+        >
+          <div className={cs.bannerText}>
+            Hold SHIFT to scroll horizontally and SPACE BAR to pan.
+            <CloseIcon
```

Comment:
We have a "remove" icon that is very similar (except for the circle). We cannot reuse that one? 
@happyimadesignr do we have a reasoning to use on or the other?